### PR TITLE
add helper to handle log sends

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers i made this to help me debug some log sends. basically save your log send as log.txt in desktop then it'll output log.json. also has a function you can hand edit to filter down the store so your output isn't too noisy

### CODE REVIEW

File: desktop/npm-helper.js [hunk 1]
```diff
@@ -190,6 +195,70 @@ function fixupSymlinks () {
   }
 }
 
+// Edit this function to filter down the store in the undiff log
+function storeFilter (store) {
+  return store
+  // return {serverStarted: store.tracker.serverStarted}
```
Comment (chrisnojima) [inline desktop/npm-helper.js]: just an example

**Comment by chrisnojima** [comment]
since this is a very internal dev tool i'm not adding a bunch of bells and whistles, hence the fixed log.txt / log.json thing

**Comment by chrisnojima** [comment]
added actions filter as gregor actions have a massive payload

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by MarcoPolo** [comment]
This didn't work for me for these two logs:

cbabd3057f89bf27cfd5121c
c60bc6616e1da0bfc3ec321c

**Comment by chrisnojima** [comment]
@MarcoPolo I think this is an artifact of the logs being truncated so the diff isn't starting from an empty object. I think this is unavoidable. I'm catching/logging these errors and continuing which seems to output stuff that's still useful. This allows both those logs to be output

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**